src/Cafe/GraphicPack/GraphicPack2.cpp

```diff
@@ -960,7 +960,7 @@ bool GraphicPack2::Activate()
 				auto option_upscale = rules.FindOption("upscaleMagFilter");
 				if(option_upscale && boost::iequals(*option_upscale, "NearestNeighbor"))
 					m_output_settings.upscale_filter = LatteTextureView::MagFilter::kNearestNeighbor;
-				auto option_downscale = rules.FindOption("NearestNeighbor");
+				auto option_downscale = rules.FindOption("downscaleMagFilter");
```

Comment:
This should be named "downscaleMinFilter" to match [OpenGL/Vulkan terminology](https://gdbooks.gitbooks.io/legacyopengl/content/Chapter7/MinMag.html). Internally the code is just abusing `LatteTextureView::MagFilter` but it should probably be its own enum for Graphic Packs.

Reply:
Ah okay. The reason I went with downscaleMagFilter is because that's what I saw in all of the community graphic packs.